packages/astro/src/transitions/vite-plugin-transitions.ts

```diff
@@ -19,6 +24,11 @@ export default function astroTransitions(): vite.Plugin {
 				export { default as ViewTransitions } from "astro/components/ViewTransitions.astro";
 			`;
 			}
+			if (id === resolvedVirtualClientModuleId) {
+				return `
+				export * from "astro/transitions/router";
```

Comment:
Shouldn't this be `astro/transitions/router/client`? If not, why?

Reply:
Rational: in the future, there might be other transformation client stuff  besides the router